Automattic/wp-calypso
# WIP | Launchpad: Add/launchpad email banner tests

#### Summary
This PR lays the groundwork for testing the `step-content.tsx` component and adds tests for the `email-validation-banner.tsx`

#### Testing Instructions

Run `yarn test-client launchpad`. You should see all tests are passing:

<img width="1161" alt="image" src="https://user-images.githubusercontent.com/20927667/199668167-34ca298b-fdf5-48b1-ac1c-102026004442.png">


#### Notes
There are some tests that I wasn't able to add when it comes to testing the resending of the email - testing the success and error notices. Currently, I am unable to access the resend email button inside my test files. I've asked a question in [slack](p1667418042437409-slack-C03N25JPCE4) 



#### Pre-merge Checklist






Related to #69551
